gossip/src/cluster_info.rs

```diff
@@ -962,6 +962,23 @@ impl ClusterInfo {
         }
     }
 
+    pub fn push_restart_last_voted_fork_slots(&self, update: &[Slot], last_vote_bankhash: Hash) {
```

Comment:
`update` is pretty ambiguous here.
can we name the arg `fork`?

Reply:
Done.